src/main/resources/com/google/api/codegen/java/fragment.snip

```diff
@@ -12,7 +12,7 @@
     /*
      * PRE-REQUISITES:
      * ---------------
-     * 1. If not already done, enable the {@ServiceName} API and check quota for your project at
+     * 1. If not already done, enable the {@ServiceName} API and check the quota for your project at
```

Comment:
Hmm. This file is not being used. It should be deleted. Are you good with the PR apart from this? If so, I'll just delete it and merge.